src/documentation/examples/Sections/ProductDetailSection.tsx

```diff
@@ -61,22 +62,25 @@ export default class App extends Vue {
           { src: src2, type: "image" },
         ]}
         propertyList={propertyList}
-        foldableContentList={{
-          Delivery: (
-            <ul class="ui-list-disc ui-px-10 ui-py-4">
-              <li>mechanical lock</li>
-              <li>camera with integrated face recognition</li>
-              <li>wireless numpad</li>
-              <li>operation manual</li>
-            </ul>
-          ),
-          "Installation instructions": <p>Fix it at the door. Ready.</p>,
-          Compatibility: (
-            <ul class="ui-list-disc ui-px-10 ui-py-4">
-              <li>Google Home</li>
-              <li>Amazon Alexa</li>
-              <li>Bosch Smart Home</li>
-            </ul>
+        scopedSlots={{
+          additionalContent: () => (
+            <div class="ui-mx-auto ui-w-full ui-px-6 ui-mb-4">
+              <Accordion dark title="Delivery">
+                <ul class="ui-list-disc ui-px-10 ui-py-4">
+                  <li>mechanical lock</li>
+                  <li>camera with integrated face recognition</li>
+                  <li>wireless numpad</li>
+                  <li>operation manual</li>
+                </ul>
+              </Accordion>
+              <Accordion dark title="Compatibility">
```

Comment:
"Compatibility" is already in one of the property lists for this example. I propose a different list here:
**Surface options**
- steel
- silver
- gilded